article/app/controllers/LiveBlogController.scala

```diff
@@ -185,20 +185,32 @@ class LiveBlogController(
                 blog,
                 blocks,
                 pageType,
+                None,
```

Comment:
These unnamed params are so fiddly to read!

Reply:
Fixed :)